src/sounds.cpp

```diff
@@ -381,9 +381,12 @@ void sounds::process_sound_markers( player *p )
     bool is_deaf = p->is_deaf();
     const float volume_multiplier = p->hearing_ability();
     const int weather_vol = get_weather().weather_id->sound_attn;
-    for( const auto &sound_event_pair : sounds_since_last_turn ) {
-        const tripoint &pos = sound_event_pair.first;
-        const sound_event &sound = sound_event_pair.second;
+    for( std::size_t i = 0; i < sounds_since_last_turn.size(); i++ ) {
```

Comment:
Probably going to need to tell clang-tidy to not tell you to turn this into a range-based for.